features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ExploreDatabaseMigrations.kt

```diff
@@ -178,5 +178,9 @@ class ExploreDatabaseMigrations {
                     )
                 }
             }
+
+        val migration2To3 = object : Migration(2, 3) {
+            override fun migrate(database: SupportSQLiteDatabase) { }
+        }
```

Comment:
Is the migration empty because typically, the APK would be including a preloaded v3 database?

Reply:
We need an empty migration because the app will complain about changed schema without a version bump, and then we need a migration for a bump.

It is empty because the new database will already include `denominationsType` field and adding this migration here  will crash the app.

The only issue will arise briefly when db-v1 user updates to the DashSpend build, but then they'll have their db updated to preloaded v2 or downloaded v2.